src/main/java/dev/xpple/simplewaypoints/commands/WaypointCommand.java

```
@@ -205,6 +209,18 @@ private static int list(FabricClientCommandSource source, boolean current) {
         return count[0];
     }
 
+    private static int get(FabricClientCommandSource source, String name) {
+        Waypoint waypoint = API.getWorldWaypoints(API.getWorldIdentifier(source.getClient())).get(name);
+
+        if (waypoint == null) {
+            source.sendFeedback(Component.translatable("commands.sw:waypoint.invalidWaypointName", name));
+            return 0;
+        }
```

Please throw exceptions instead of returning zero.